tests/dragonfly/replication_test.py

```diff
@@ -29,7 +29,7 @@
     (8, [2, 2, 2, 2], dict(keys=4_000, dbcount=4)),
     (4, [8, 8], dict(keys=4_000, dbcount=4)),
     (4, [1] * 8, dict(keys=500, dbcount=2)),
-    (1, [1], dict(keys=100, dbcount=2)),
+    (1, [1], dict(keys=10_000, dbcount=2)),
```

Comment:
MMM yes this is how I found the bug.. what do you suggest?

Reply:
Wow. I suggest hunting bugs locally or on separate branches, because this will most likely time out the test. 

We won't cover all of the difficult bugs on CI tests anyways, we just wan't to make sure that nothing essential broke between commits